### PR TITLE
Reduce gas limit of EVM call and remove the last 2 instructions of XCM

### PR DESCRIPTION
**Pull request by chrisli30**
CleanShot 2023-09-17 at 20 33 09

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frocstar.astar.network#/explorer/query/0xafe10a8d950e7b62b365fcd664cd9581f4e9643c15c3afa9651be65018106d41

## Change
gasLimit: 201,596
XCM Instructions: 4

## Result
refTime: 
9,367,272,555
proofSize: 
320,259

### CODE REVIEW

File: src/astar/rocstar.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ import { scheduleTask } from './arthswap';
 // const createTaskPayload = (astarApi) => astarApi.tx.system.remarkWithEvent('Hello world!');
 
 const createTaskPayload = (astarApi) => astarApi.tx.ethereumChecked.transact({
-    gasLimit: 3000000,
+    gasLimit: 201596,
```
Comment (chrisli30) [inline src/astar/rocstar.js:18]: The huge weight is caused by inaccurate gasLimit of the smart contract.

File: src/astar/rocstar.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ import { scheduleTask } from './arthswap';
...
     target: '0xA17E7Ba271dC2CC12BA5ECf6D178bF818A6D76EB',
-    value: new BN('100000000000000000'),
+    value: new BN('10000000000000000'),
```
Comment (chrisli30) [inline src/astar/rocstar.js:20]: I changed this value to 0.01 RSTR as it’s burning my balance.

File: src/astar/arthswap.js [hunk 3]
```diff
@@ -159,17 +164,6 @@ export const scheduleTask = async ({
...
-            {
-                DepositAsset: {
-                    assets: { Wild: { AllCounted: 1 } },
-                    maxAssets: 1,
-                    beneficiary: {
-                        parents: 1,
-                        interior: { X1: { AccountId32: { network: null, id: u8aToHex(keyringPair.addressRaw) } } },
-                    },
-                },
-            },
```
Comment (chrisli30) [inline src/astar/arthswap.js:172]: Had to remove the last two instructions to make it work.

**Review by imstar15 — APPROVED** [review]
Great! Looks good to me!

**Pull request merged**